x/ccv/consumer/keeper/migration.go

```diff
@@ -0,0 +1,83 @@
+package keeper
+
+import (
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	consumertypes "github.com/cosmos/interchain-security/v2/x/ccv/consumer/types"
+	ccvtypes "github.com/cosmos/interchain-security/v2/x/ccv/types"
+)
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	ccvConsumerKeeper     Keeper
+	ccvConsumerParamSpace paramtypes.Subspace
+}
+
+// NewMigrator returns a new Migrator.
+func NewMigrator(ccvConsumerKeeper Keeper, ccvConsumerParamSpace paramtypes.Subspace) Migrator {
+	return Migrator{ccvConsumerKeeper: ccvConsumerKeeper, ccvConsumerParamSpace: ccvConsumerParamSpace}
+}
+
+// Migratev1p2p0MultidenTov2 migrates a consumer from v1.2.0-multiden to v2.0.0.
+func (m Migrator) Migratev1p2p0MultidenTov2(ctx sdk.Context) error {
```

Comment:
Remove this function as is not needed. State migration cares about consensus versions and not semver. As both v1.2.0-multiden and v1.0.0 are on the same consensus version (i.e., `1`), we just need `Migratev1Tov2`

Reply:
@mpoke see https://github.com/cosmos/interchain-security/pull/975/commits/a235a1ab0c5b8514ec791fcd9b638d5cb9e1387c. I think this is the best way to deal with consensus version for just the consumer. Moving forward we'll know to increment that number when migrations are required

Reply:
But as far as your original comment, the methods were removed 👍